src/main/java/juno/command/MarkCommand.java

```diff
@@ -0,0 +1,92 @@
+package juno.command;
+
+import java.util.ArrayList;
+
+import juno.manager.FileManager;
+import juno.manager.TaskManager;
+import juno.manager.exception.TaskManagerException;
+import juno.task.Task;
+
+
+/**
+ * A class to mark or unmark a task as done or not done (with a X).
+ * Updates the status of a specified task based on the user input.
+ * Updates the file tasks.json() after the <code>runCommand()</code> is executed.
+ */
+
+public class MarkCommand extends Command {
+    private TaskManager taskManager;
+    private FileManager fileManager;
+    private String userInput;
+    private ArrayList<Task> tasks;
+    private boolean markAsDone;
```

Comment:
Perhaps the name used can be made to sound more like a boolean? e.g. isMarkedAsDone